html-web-page/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+  <h1>Jelly of the Month Club</h1>
+  <h2>Make all your friends jelly of your stock of jelly!</h2>
+  <p>
+    Lorem ipsum dolor sit amet consectetur adipisicing elit. Adipisci vero quam doloremque voluptas maxime praesentium consequuntur corrupti nostrum molestiae ut eos illum veritatis autem expedita velit, enim suscipit minus harum?
+    Lorem ipsum dolor sit amet consectetur adipisicing elit. Sint ducimus reprehenderit voluptas beatae non dolorum facere at molestiae similique quod cumque saepe officiis ipsa sit mollitia, nulla perferendis corrupti labore?
+  </p>
+
+  <table>
+   <tr>
+     <th></th>
+     <th>Basic Plan</th>
+     <th>Deluxe Plan</th>
+     <th>Jelly Giant Plan</th>
+   </tr>
+   <tr>
+     <td>Three new curated jellies per month</td>
+     <th>&check;</th>
+     <th>&check;</th>
+     <th>&check;</th>
+   </tr>
+   <tr>
+     <td>Monthly jellies come with commemorative mason jars</td>
+     <th></th>
+     <th>&check;</th>
+     <th>&check;</th>
+   </tr>
+   <tr>
+     <td>Free bonus jar of preserves bi-monthly</td>
+     <th></th>
+     <th>&check;</th>
+     <th>&check;</th>
+   </tr>
+   <tr>
+     <td>Free yearly tickets to Jelly Con</td>
+    <th></th>
+    <th></th>
+    <th>&check;</th>
+   </tr>
+   <tr>
+     <td>Monthly Pricing</td>
+     <th>$30</th>
+     <th>$50</th>
+     <th>$60</th>
+   </tr>
```

Comment:
This should be in a `tbody`. Remember to go back and reference previous exercises if you're unsure about how to do something, you just got this feedback on the last exercise as well.